app/styles/globals.css

```diff
@@ -0,0 +1,16 @@
+html,
```

Comment:
Is this file default generated or is there custom work here?

Reply:
if it's generated, could we instead have instructions for generating it, rather than have the already generated artifact in the template? that would allow more flexibility to receive updates on the generation process

Reply:
this was default generated!

Reply:
It was default generated but I deleted the classes that I'm not using and I added a class for the header landmark. @lorenyu could I get some clarification regarding the kind of instructions for generating it that you're thinking of?

Reply:
@lorenyu I think that's a great idea! Good point regarding the potential issues that locking the generated files could create. I'm not the most infra-savvy but would it be possible to set up a CI that checks for code changes to `Next.js` and automate integration with our repo? And if there are conflicts with their updates and our existing custom changes, then we address and resolve on an ongoing basis? p.s. I added a list of the auto-generated files that I haven't touched yet in the PR description. 

@rocketnova sounds good :)

Reply:
@rocketnova sounds good!
@karinamzalez not sure what's possible! i don't really have a specific vision for how this looks, just raising the topic for discussion that's all. given the uncertainty, my bias is to keep things simple and do things manually i.e. go with the current approach of committing the files, but having some sort of place where we're documenting the changes we made and why we made those changes.

Reply:
@lorenyu sounds good -- I'll document the changes/reasoning in a more detailed manner in the README.

Reply:
Issue created: #7

Reply:
@karinamzalez I unresolved this comment because I wanted to make sure you got the changes and reasoning documented in `app/README.md`. It doesn't look like this PR has that yet. 👀